monkit.lock

```diff
@@ -0,0 +1,18 @@
+storj.io/storj/satellite/accounting/tally."total.bytes" IntVal
+storj.io/storj/satellite/accounting/tally."total.inline_bytes" IntVal
+storj.io/storj/satellite/accounting/tally."total.inline_segments" IntVal
+storj.io/storj/satellite/accounting/tally."total.objects" IntVal
+storj.io/storj/satellite/accounting/tally."total.remote_bytes" IntVal
+storj.io/storj/satellite/accounting/tally."total.remote_segments" IntVal
+storj.io/storj/satellite/accounting/tally."total.segments" IntVal
```

Comment:
This is not the correct lock, it should be `storj.io/storj/satellite/accounting."total.segments"`, similarly all others.

Reply:
@egonelbre when you say similarly all others do you mean all of the lines with `storj.io/storj/satellite/accounting/tally` should be changed to just `storj.io/storj/satellite/accounting` or that the `storj.io/storj/satellite/repair/checker` should also be changed to `storj.io/storj/satellite/repair`?

Reply:
@egonelbre can you elaborate on what is not correct about this?

Reply:
The `monAccounting` has a different prefix specified than the package it is in. See the definition of `monAccounting`.

Reply:
I see what you mean. Monkit will still report those metrics with the correct key but the lockfile generation script doesn't look at the monkit scope or anything, just the AST. I don't think this should shouldn't be problematic as long as the script is consistent. It might be a bit confusing that the lockfile entry doesn't match the actual key that's reported but I'm not sure it's worth fixing right now.

Reply:
The only way I could see this being a problem is if two monkit calls with different keys produce the same lockfile entry because they will be deduplicated.

Reply:
The issue is that when somebody sees that there's `monAccounting` used and deletes it and uses `mon` instead, the linter will still succeed.

Reply:
Ah... yeah okay, that's not good.

Reply:
Any suggestions other than to track which scopes calls are made from?

Reply:
Get the receiver for that call and then look at the constructor.